### DISCUSSION TITLE
LICENSE file should contain copyright holder

### DISCUSSION THREAD

**Discussion by bernhard-voelker** [abseil/abseil-cpp#1126] [General]
The license file currently says:

   Copyright [yyyy] [name of copyright owner]

IMO the placeholders should be filled, right?

**Comment by derekmauro** [comment]
That line is in a section that says "APPENDIX: How to apply the Apache License to your work."

We've actually been asked about this before, so I asked a lawyer with expertise in open source licenses to confirm. The response I got was:
> Nobody should be changing the Apache license appendix to specify a copyright holder. IE the linked to line/file is part of an appendix to the License, and should *not* be modified. If you see that happening, please let [our team] know so [we] can get it fixed. The LICENSE files we have should be pristine Apache 2.0 licenses.

**Reply by bernhard-voelker** [reply]
Interesting - thanks for clarification.
